Assets/Scripts/MapGen/WorldMapCamera.cs

```diff
@@ -9,19 +10,42 @@ public class WorldMapCamera : MonoBehaviour
 
         new Camera camera;
 
+        void OnEnable()
+        {
+            GlobalEvent.OnRoomLoaded += OnRoomLoaded;
+        }
+
+        void OnDisable()
+        {
+            GlobalEvent.OnRoomLoaded -= OnRoomLoaded;
+        }
+
         void Awake()
         {
             camera = GetComponent<Camera>();
+            camera.enabled = false;
         }
 
         void Update()
         {
             // TODO: REMOVE THIS IN FAVOR OF PLAYER CONTROLLER + GLOBAL EVENT
             if (Input.GetButtonDown("Submit"))
             {
-                camera.depth *= -1;
+                camera.enabled = !camera.enabled;
```

Comment:
also [changed the camera depth](https://github.com/MitchellCoon/MetroidvaniaGameJamFeb23/pull/21/files#diff-b37bc886441c6295217038a7075bab5628a2fa2a1181d3495e90010a48023d7aR64) so that it appears on top by default